File: base/src/main/java/com/windscribe/vpn/api/response/TicketResponse.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response;


import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Keep
public class TicketResponse {

    @SerializedName("status")
    @Expose
    private String status;

    @SerializedName("ticket_id")
    @Expose
    private String ticketId;

    public String getTicketId() {
        return ticketId;
    }
}

File: base/src/main/java/com/windscribe/vpn/api/response/TicketResponse.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Keep
class TicketResponse {
    @SerializedName("status")
    @Expose
    val status: String? = null

    @SerializedName("ticket_id")
    @Expose
    val ticketId: String? = null
}
